management/src/main/java/org/ehcache/management/config/EhcacheStatisticsProviderConfiguration.java

```diff
@@ -22,63 +22,14 @@
 /**
  * @author Ludovic Orban
  */
-public class EhcacheStatisticsProviderConfiguration implements StatisticsProviderConfiguration {
-
-  private long averageWindowDuration;
-  private TimeUnit averageWindowUnit;
-  private int historySize;
-  private long historyInterval;
-  private TimeUnit historyIntervalUnit;
-  private long timeToDisable;
-  private TimeUnit timeToDisableUnit;
+public class EhcacheStatisticsProviderConfiguration extends DefaultStatisticsProviderConfiguration {
```

Comment:
Because you can have several stat configuration : one per provider actually.

EhcacheStatisticsProviderConfiguration is one.

So when configuring through XML (see `<statisticConfiguration>`), you have to optionally specify the provider you want this config to be applied to.